pom.xml

```diff
@@ -84,6 +84,24 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
+        <groupId>org.glassfish</groupId>
+        <artifactId>javax.json</artifactId>
+        <!-- Old version until this issue is resolved: https://github.com/ImmobilienScout24/illegal-transitive-dependency-check/issues/30 -->
+        <version>1.0.4</version>
+      </dependency>
+      <dependency>
+        <groupId>org.mock-server</groupId>
+        <artifactId>mockserver-netty</artifactId>
```

Comment:
Under discussion - addding @rsynek  - wether this dependency it's worth it's weight. I don't think it is.

Reply:
For reference, this is the dependency tree of mockserver-netty:
```
[INFO] +- org.mock-server:mockserver-netty:jar:5.3.0:test
[INFO] |  +- org.mock-server:mockserver-client-java:jar:5.3.0:test
[INFO] |  +- org.mock-server:mockserver-core:jar:5.3.0:test
[INFO] |  |  +- javax.servlet:javax.servlet-api:jar:3.1.0:test
[INFO] |  |  +- io.netty:netty-codec-socks:jar:4.1.17.Final:test
[INFO] |  |  +- com.jcraft:jzlib:jar:1.1.3:test
[INFO] |  |  +- org.bouncycastle:bcmail-jdk15on:jar:1.58:test
[INFO] |  |  +- org.bouncycastle:bcpkix-jdk15on:jar:1.58:test
[INFO] |  |  +- org.apache.velocity:velocity-engine-scripting:jar:2.0:test
[INFO] |  |  +- org.apache.velocity:velocity-engine-core:jar:2.0:test
[INFO] |  |  +- org.skyscreamer:jsonassert:jar:1.2.3:test
[INFO] |  |  |  \- org.json:json:jar:20090211:test
[INFO] |  |  +- com.github.java-json-tools:json-schema-validator:jar:2.2.8:test
[INFO] |  |  |  +- com.github.java-json-tools:json-schema-core:jar:1.2.8:test
[INFO] |  |  |  |  +- org.mozilla:rhino:jar:1.7R4:test
[INFO] |  |  |  |  +- com.github.fge:jackson-coreutils:jar:1.8:test
[INFO] |  |  |  |  |  \- com.github.fge:msg-simple:jar:1.1:test
[INFO] |  |  |  |  |     \- com.github.fge:btf:jar:1.2:test
[INFO] |  |  |  |  \- com.github.fge:uri-template:jar:0.9:test
[INFO] |  |  |  +- javax.mail:mailapi:jar:1.4.3:test
[INFO] |  |  |  +- joda-time:joda-time:jar:2.9.7:test
[INFO] |  |  |  +- com.googlecode.libphonenumber:libphonenumber:jar:8.0.0:test
[INFO] |  |  |  +- com.google.code.findbugs:jsr305:jar:3.0.1:test
[INFO] |  |  |  \- net.sf.jopt-simple:jopt-simple:jar:5.0.3:test
[INFO] |  |  +- org.apache.commons:commons-text:jar:1.1:test
[INFO] |  |  \- commons-collections:commons-collections:jar:3.2.2:test
[INFO] |  +- io.netty:netty-buffer:jar:4.1.17.Final:test
[INFO] |  +- io.netty:netty-codec:jar:4.1.17.Final:test
[INFO] |  +- io.netty:netty-codec-http:jar:4.1.17.Final:test
[INFO] |  +- io.netty:netty-common:jar:4.1.17.Final:test
[INFO] |  +- io.netty:netty-handler:jar:4.1.17.Final:test
[INFO] |  +- io.netty:netty-transport:jar:4.1.17.Final:test
[INFO] |  |  \- io.netty:netty-resolver:jar:4.1.17.Final:test
[INFO] |  +- org.bouncycastle:bcprov-jdk15on:jar:1.58:test
[INFO] |  +- janino:janino:jar:2.5.10:test
[INFO] |  \- org.mock-server:mockserver-logging:jar:5.3.0:test
```

Reply:
We'd like to have the mocknetty-server dependency removed - even if that impacts test coverage negatively. The quality of that dependency isn't high enough:
- bloat (too many dependencies)
- very old unmaintained dependencies (velocity, bcprov-jdk15, ...)